libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.h

```diff
@@ -515,12 +515,20 @@ Use this method to stop/clear any authentication which is has already been start
 @param completionBlock The completion block is called with YES if a session was cleared successfully. 
 */
 - (void)stopCurrentAuthentication:(nullable void (^)(BOOL))completionBlock;
+
 /**
  Forces a logout from the current account, redirecting the user to the login process.
  This throws out the OAuth refresh token.
  */
 - (void)logout;
 
+/**
+ Forces a logout from the current account, redirecting the user to the login process.
+ This throws out the OAuth refresh token.
+ @param reason The reason that log out was initiated.
+ */
+- (void)logout:(SFLogoutReason)reason;
```

Comment:
I am actually a little surprised iOS didn't have a logout without user param until now.  Maybe that's because Android has (too) many overloads :/